tests/visual/checkboxradio/checkboxradio.html

```diff
@@ -0,0 +1,69 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>jQuery UI Button - Checkboxes</title>
+	<link rel="stylesheet" href="../../../themes/base/all.css">
+	<script src="../../../external/jquery/jquery.js"></script>
+	<script src="../../../ui/core.js"></script>
+	<script src="../../../ui/widget.js"></script>
+	<script src="../../../ui/button.js"></script>
+	<script src="../../../ui/checkboxradio.js"></script>
+	<link rel="stylesheet" href="../demos.css">
+	<script>
+	$(function() {
+		var checkboxes = $( "form input" ).checkboxradio();
+
+		$( ".controls input, .controls select" ).on( "change keyup", function(){
+			var option = $( this ).attr( "name" ),
+				value = $( this ).val();
+
+			if( $( this ).is( "[type=checkbox] ") ) {
```

Comment:
Missing space after `if`.

Makes me wonder how we can apply our tool chain to embedded scripts... SublimeLinter manages that.